concepticondata/conceptlists/Ray-1895-26.tsv

```diff
@@ -0,0 +1,27 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Ray-1895-26-1	1	Bird	937	BIRD
```

Comment:
Note, @lannin, that I chanded the ID from Ray-1895-1 to Ray-1895-26-1, as the beginning is always the name of the file.